### PR TITLE
FND-85 Implement discover/candidate screen

### PR DESCRIPTION
**Pull request by HuyCao3110**

### CODE REVIEW

**Comment by snowynguyen** [comment]
next time please write the PR carefully and wait for another member's approval

File: frontend/src/app/dashboard/profile/page.tsx
Comment (snowynguyen) [inline frontend/src/app/dashboard/profile/page.tsx:1]: was there a reason to delete this file?

**Review by snowynguyen — COMMENTED** [review]
thanks for the good work
next time pls commit more frequently and elaborate on the changes you made via commit messages

File: frontend/src/app/dashboard/profile/page.tsx
Comment (TheDLCrimson) [reply frontend/src/app/dashboard/profile/page.tsx:1]: It was removed to .../dashboard/discover/candidate

**Comment by TheDLCrimson** [comment]
image

@HuyCao3110 dont add this in when you dont use it

**Pull request merged**